src/components/forms/PatientForm.tsx

```diff
@@ -72,61 +89,98 @@ const PatientForm: FunctionComponent<PatientFormProps> = ({ patient, isEditForm
 
   return (
     <Form className='mt-4' onSubmit={async e => await handleSubmit(e)}>
+      <Row className='my-3'>
+        <Form.Group as={Col} className='mt-2'>
+          <Form.Label><FontAwesomeIcon icon={['fas', 'id-card']} /> {t('form.lastName')}</Form.Label>
+          <Form.Control
+            name='lastname'
+            type='text'
+            value={patientState.lastname}
+            onChange={handleChange}
+          />
+        </Form.Group>
+
+        <Form.Group as={Col} className='mt-2'>
+          <Form.Label><FontAwesomeIcon icon={['fas', 'id-card']} /> {t('form.firstName')}</Form.Label>
+          <Form.Control
+            name='firstname'
+            type='text'
+            value={patientState.firstname}
```

Comment:
I think I've mostly seen that in France, but I dislike it, that lastname before the firstname thing. It feels so unnatural.
But whatever you prefer